Buildings/Fluid/Storage/StratifiedEnhanced.mo

```diff
@@ -52,6 +52,11 @@ The model requires at least 4 fluid segments. Hence, set <code>nSeg</code> to 4
 </html>", revisions="<html>
 <ul>
 <li>
+January 25, 2015, by Filip Jorissen:<br/>
```

Comment:
@mwetter in that case the comment should be updated since ```tau``` is no longer propagated but set to 0?